### DISCUSSION TITLE
How to highlight whitespace characters in matches?

### DISCUSSION THREAD

**Discussion by molleweide** [ggandor/lightspeed.nvim#93] [Q&A] [answered]
In target `ab` where `b` = whitespace should I expect it to be highlighted somehow, because it is not for me.

**Comment by ggandor** [comment, marked as answer]
By default, no, but you can set `require('lightspeed').opts.substitute_chars[' '] = <whatever you want spaces in matches to be displayed as>`. `newline` is the only one for which such a substitute char is set by default (`¬`).

**Reply by molleweide** [reply]
aaaah dude thanks for pointing this out. I haven't even started looking at the configuration possibilities yet. Now's the time!!
